render/template_helpers.go

```diff
@@ -0,0 +1,84 @@
+package render
+
+import (
+	"encoding/json"
+	"html/template"
+	"path/filepath"
+	"sync"
+
+	"github.com/gobuffalo/tags"
+)
+
+var mu = &sync.Mutex{}
```

Comment:
Can we rename this to something like `assetMutex`? That way we know what it's locking down?